Sources/Utility/PkgConfig.swift

```diff
@@ -282,17 +282,24 @@ struct PkgConfigParser {
         }
         
         var it = line.characters.makeIterator()
+        // Indicates if we're in a quoted fragment, we shouldn't append quote.
+        var inQuotes = false
         while let char = it.next() {
-            if char == "\\" {
+            if char == "\"" {
+                inQuotes = !inQuotes
+            } else if char == "\\" {
                 if let next = it.next() {
                     fragment.append(next)
                 }
-            } else if char == " " {
+            } else if char == " " && !inQuotes {
                 saveFragment()
             } else {
                 fragment.append(char)
             }
         }
+        guard !inQuotes else {
+            throw PkgConfigError.parsingError("Text ended before matching quote was found in line: \(line) file: \(pcFile.asString)")
```

Comment:
This is a good check, but we also are doing something different than pkg-config on cases where the quotes are inside the argument, right?